### PR TITLE
Fix common languages parsing and add tests

### PR DESCRIPTION
**Pull request by tgalopin**
This PR adds tests for a lot of common languages and countries in order to ensure parsing works properly. Some issues appeared and I fixed them here too.

### CODE REVIEW

File: lib/app.js [hunk 1]
```diff
@@ -70,11 +70,22 @@ function parseFields ($) {
   const downloads = additionalInfo.find('div.content[itemprop="numDownloads"]').text().trim();
   let installs = downloads.split(' - ');
```
Comment (facundoolano) [inline lib/app.js]: can you move all these split / length checks to a helper function?

**Comment by facundoolano** [comment]
Looks good, I left a comment about factoring the installs split logic into a function. Fix that and I'll merge.

Thanks!

**Comment by tgalopin** [comment]
Updated

**Comment by facundoolano** [comment]
thanks@

**Pull request merged**